<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>TeacherWindow</class>
    <widget class="QWidget" name="TeacherWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>780</width>
                <height>454</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QFrame" name="frame">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>20</y>
                    <width>671</width>
                    <height>401</height>
                </rect>
            </property>
            <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
                <enum>QFrame::Raised</enum>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>671</width>
                        <height>401</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QPushButton" name="add_programmes_button">
                                    <property name="text">
                                        <string>添加课程</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="delete_programmes_button">
                                    <property name="text">
                                        <string>删除课程</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_2">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="add_grade_button">
                                    <property name="text">
                                        <string>添加学生成绩</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_4">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="exit_button">
                                    <property name="text">
                                        <string>退出登录</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="Line" name="line_6">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QTableView" name="student_table_view"/>
                    </item>
                </layout>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
